whelk-core/src/main/groovy/whelk/JsonValidator.groovy

```diff
@@ -38,16 +40,29 @@ class JsonValidator {
     }
 
     Set validateAll(Map map) {
```

Comment:
This could be `List<String>` which would return errors in "logical" order (traversal order).